files/chance/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "chance",
+  "repo": "victorquinn/chancejs",
+  "files": {
+    "basePath": "dist/"
```

Comment:
I just pushed a new version of Chance with a `dist` folder containing a minified version of Chance so this should theoretically work now!